packages/neuron-wallet/src/controllers/asset-account.ts

```diff
@@ -0,0 +1,109 @@
+import AssetAccount from "models/asset-account"
+import Transaction from "models/chain/transaction"
+import AssetAccountService from "services/asset-account-service"
+import { ServiceHasNoResponse } from "exceptions"
+import { ResponseCode } from "utils/const"
+import AddressService from "services/addresses"
+import NetworksService from "services/networks"
+import AddressGenerator from "models/address-generator"
+import { AddressPrefix } from "@nervosnetwork/ckb-sdk-utils"
+
+export interface GenerateCreateAssetAccountTxParams {
+  walletID: string
+  tokenID: string
+  fullName: string
+  symbol: string
+  decimal: string
+  feeRate: string
+  fee: string
+}
+
+export interface SendCreateAssetAccountTxParams {
+  walletID: string
+  assetAccount: AssetAccount
+  tx: Transaction
+  password: string
+}
+
+export interface UpdateAssetAccountParams {
+  id: number
+  walletID?: string
```

Comment:
The wallet id and token id cannot be changed.

Reply:
updated